tests/fixtures/mock_halpy.py

```diff
@@ -25,6 +25,8 @@
 class TestBot(HalpyBOT):
     """A Fully-Lobotimized Version of HalpyBOT"""
 
+    __test__ = False
+
```

Comment:
what?

Reply:
As the class starts with the word "Test", Pytest attempts to load it as a unit test. This marker disables that. See: https://stackoverflow.com/questions/62460557/cannot-collect-test-class-testmain-because-it-has-a-init-constructor-from